src/reformatters/common/download.py

```diff
@@ -77,6 +77,33 @@ def http_store(base_url: str) -> obstore.store.HTTPStore:
     )
 
 
+@functools.cache
+def s3_store(
+    bucket_url: str, region: str, skip_signature: bool = True
```

Comment:
This defaults to assuming you are working with public buckets and it would be opt in to use provisioned credentials, but we could invert.